### PR TITLE
build/cy4390x : Add defconfig and make file for st_things.

### PR DESCRIPTION
**Pull request by uvasistha-samsung**
Add defconfig and make file to build TizenRT and flash binary for st_things module for cy43909.

Signed-off-by: uvasistha-samsung utkarsh.m@samsung.com

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [666c774991c1c98a6da1587f99a5b4cba2942776] - Code Rule Check OK.

**Comment by sunghan-chang** [comment]
@SenthilKumarGS Thank you for updating and that's ok. Please let me know when you verify.

File: build/configs/cy4390x/st_things/defconfig [hunk 1]
```diff
@@ -0,0 +1,1202 @@
...
+# Database
+#
+# CONFIG_ARASTORAGE is not set
+
+#
+# Memory Management
+#
+# CONFIG_DISABLE_REALLOC_NEIGHBOR_EXTENSION is not set
+# CONFIG_MM_SMALL is not set
+CONFIG_MM_REGIONS=1
```
Comment (jeongchanKim) [inline build/configs/cy4390x/st_things/defconfig]: Please check this commit 
(https://github.com/Samsung/TizenRT/commit/a0c022e5f5f021af241b1ce3eb6dc67eeb01afb7)
CONFIG_MM_NHEAPS=1
CONFIG_RAM_REGIONx_START
CONFIG_RAM_REGIONx_SIZE
are needed.

**Comment by Hyunjun85** [comment]
@SenthilKumarGS Hi.
As I know, this is resolved by Mr. Utkarsh.
Could you please confirm this?

**Comment by Hyunjun85** [comment]
@uvasistha-samsung Could you please update status?
@SenthilKumarGS

**Comment by SenthilKumarGS** [comment]
Hi @uvasistha-samsung ,
Please update the status of this task.

**Comment by Hyunjun85** [comment]
It should be closed why It has been opened too long time and there is no additional works.

**Pull request closed without merging**